documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -2206,9 +2209,12 @@ You can set the following values:
 |Specifies the mode that the connector uses to take snapshots of a captured table.
 You can set the following values:
 
-`initial`:: The snapshot includes the structure and data of captured tables.
+`initial`:: The snapshot includes the structure and data of a captured table.
```

Comment:
```suggestion
`initial`:: The snapshot includes the structure and data of the captured tables.
```
We shouldn't speak in terms of a singular table here because that's not accurate.  We should retain the plural form.